packages/types/custom/core.ts

```diff
@@ -228,7 +228,21 @@ export interface GfiLayerConfiguration {
   format?: string
 }
 
-export interface GeoLocationConfiguration extends PluginOptions {
+export interface LayerBoundPluginOptions extends PluginOptions {
```

Comment:
A changelog entry for these changes is missing

Reply:
https://github.com/Dataport/polar/pull/10/commits/ce793407c2aa1a7c9550c59919c7128127d3442b